man/fi_av.3.md

```diff
@@ -285,6 +292,14 @@ struct fi_av_attr {
   targets distributed applications on large fabrics and allows for
   highly-optimized storage of remote endpoint addressing.
 
+- *FI_AV_USER_ID*
+: Indicates that the user will be associating user-defined IDs with
+  address vector. Specifying this flag during address vector creation
+  allows providers to optimize the address vector accordingly.
+
+  If users require FI_AV_AUTH_KEY and FI_AV_USER_ID support, using the
```

Comment:
Is it `and` or `or`? Please cross check the description of the FI_AV_USER_ID flag for the insert calls below.

Reply:
Should be or.